README.md

```diff
@@ -29,7 +29,14 @@ make deploy IMG=<some-registry>/terraclassic.operator:tag
 3. Install Instances of Custom Resources:
 
 ```sh
-kubectl apply -f config/samples/
+kubectl apply -f config/test-samples/
+```
+
+4. Expose service for two sample network
+
+```sh
+kubectl expose pod atreides
```

Comment:
I generally dont like the use of expose. Its a "lazy" approach to creating services for your pods that often leads to problems down the road as you no longer have control of your "desired state", which makes the "actual state" in the k8s cluster almost impossible to reason about without connecting kubectl and describing the running service instance, which in turn slows down development

Reply:
I have added service creation into terrad node controller. It should be automatic now.